apps/web/screens/Organization/Syslumenn/Auctions.tsx

```diff
@@ -236,64 +611,183 @@ const Auctions: Screen<AuctionsProps> = ({ organizationPage, namespace }) => {
+                      KNOWN_LOCATIONS.includes(auction.location) &&
+                      ' - ' + auction.location}
+                  </Text>
+                  <Tag disabled>{auction.office}</Tag>
+                </Box>
+                <Box>
+                  <Text variant="h3">{auction.lotName}</Text>
+
+                  {/* Real Estate link */}
+                  {auction.lotId &&
+                    auction.lotType === LOT_TYPES.REAL_ESTATE && (
+                      <LotLink
+                        prefix={n(
+                          'auctionRealEstateNumberPrefix',
+                          'Fasteign nr. ',
+                        )}
+                        linkText={auction.lotId}
+                        href={`https://www.skra.is/default.aspx?pageid=d5db1b6d-0650-11e6-943c-005056851dd2&selector=streetname&streetname=${auction.lotId}&submitbutton=Leita`}
+                      />
+                    )}
+
+                  {/* Vehicle link */}
+                  {auction.lotId && auction.lotType === LOT_TYPES.VEHICLE && (
+                    <LotLink
+                      prefix={n('auctionVehicleNumberPrefix', 'Bílnúmer: ')}
+                      linkText={auction.lotId}
+                      href={`https://www.samgongustofa.is/umferd/okutaeki/okutaekjaskra/uppfletting?vq=${auction.lotId}`}
+                    />
+                  )}
+
+                  {/* Aircraft link */}
+                  {auction.lotId && auction.lotType === LOT_TYPES.AIRCRAFT && (
+                    <LotLink
+                      prefix={n(
+                        'auctionAircraftNumberPrefix',
+                        'Númer loftfars: ',
+                      )}
+                      linkText={auction.lotId}
+                      href={`https://www.samgongustofa.is/flug/loftfor/loftfaraskra?aq=${auction.lotId}`}
+                    />
+                  )}
+
+                  {/* Ship link */}
+                  {auction.lotId && auction.lotType === LOT_TYPES.SHIP && (
+                    <LotLink
+                      prefix={n('auctionShipNumberPrefix', 'Númer skips: ')}
+                      linkText={auction.lotId}
+                      href={`https://www.samgongustofa.is/siglingar/skrar-og-utgafa/skipaskra/uppfletting?sq=${auction.lotId}`}
+                    />
+                  )}
+
+                  <Box
+                    alignItems="flexEnd"
+                    display="flex"
+                    flexDirection="row"
+                    justifyContent="spaceBetween"
+                    marginLeft="auto"
+                  >
+                    <Box>
+                      {auction.lotItems && (
+                        <DialogPrompt
+                          baseId={auction.lotId}
+                          title={auction.lotName}
+                          description={auction.lotItems
+                            .split('|')
+                            .join('  •  ')}
```

Comment:
@Ingimarsson 
Ideally, this would be listed as a bulleted list, but the DialogPrompt only supports text. We should try to find an alternative dialog to accomplish having a bulleted list (similar to what you did in stafraent-island page).

Reply:
We'll use the current implementation in the first version of this new auctions page. We'll then iterate on the UI in follow-up PRs, if necessary. Resolving this discussion.